src/components/dashboard/FeedList.js

```diff
@@ -229,18 +232,19 @@ const FeedList = ({
         numColumns={1}
         onEndReached={onEndReached}
         onEndReachedThreshold={onEndReachedThreshold}
-        onMomentumScrollEnd={_onScrollEnd}
+        onMomentumScrollEnd={_onScrollEnd && handleShowButton}
```

Comment:
this is wrong. it will not execute _onScrollEnd
you need
onScrollEnd = useCallbac((x) => {_onScrollEnd(x); handleShowButton()},[...])